addon/components/cf-content.js

```diff
@@ -39,103 +38,79 @@ import { parseDocument } from "ember-caluma/lib/parsers";
  * @yield {CfNavigationComponent} content.navigation
  * @yield {CfPaginationComponent} content.pagination
  */
-export default Component.extend({
-  router: service(),
-  calumaStore: service(),
+export default class CfContentComponent extends Component {
+  @service router;
+  @service calumaStore;
 
-  apollo: queryManager(),
-
-  init(...args) {
-    this._super(...args);
-
-    assert(
-      "A `documentId` must be passed to `{{cf-content}}`",
-      this.documentId
-    );
-  },
-
-  willDestroy(...args) {
-    this._super(...args);
-
-    this._teardown();
-  },
-
-  _teardown() {
-    if (this.document) this.document.destroy();
-    if (this.navigation) this.navigation.destroy();
-  },
+  @queryManager apollo;
 
   /**
    * The uuid of the document to display
    *
    * @argument {String} documentId
    */
```

Comment:
Are the arguments picked up by addon-docs even without the variable declaration?

Reply:
Yes